x-pack/plugins/security_solution/public/management/cypress/e2e/endpoint/artifacts.cy.ts

```diff
@@ -36,7 +36,7 @@ const yieldAppliedEndpointRevision = (): Cypress.Chainable<number> =>
 
 const parseRevNumber = (revString: string) => Number(revString.match(/\d+/)?.[0]);
 
-describe('Artifact pages', () => {
+describe('Artifact pages', { tags: ['@ess', '@serverless', '@brokenInServerless'] }, () => {
```

Comment:
It would be great if we could type this new property (`tags`) so that we get content assist. Can you add that to our `cypress.d.ts` file here: https://github.com/elastic/kibana/blob/054cdbaf1e030f887953681abaa72dccac633480/x-pack/plugins/security_solution/public/management/cypress/cypress.d.ts#L38-L42

Reply:
Actually - looks like `tags` is already defined via `@cypress` types so need to add to the our `cypress.d.ts` file. I do wonder if there is a way to make the type more specific in that it can suggest our "known" tags 🤔

Reply:
I did some experiments with types to be able to have suggestions, but didn't work so far, exactly because `tags` are already defined in `cypress.d.ts`, and therefore it cannot be _overriden_ by a more exact type.

one solution would be to use our own `security_tags` or something, but I'm not sure if grep can pick it up - I'll look into this, but this is definitely an open question.